notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb

```diff
@@ -0,0 +1,1425 @@
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "run_automl_pipeline:image"
+      },
+      "source": [
+        "*Example output:*\n",
+        "\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:View Training:\n",
+        "    https://console.cloud.google.com/ai/platform/locations/us-central1/training/2109316300865011712?project=759209241365\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712 current state:\n",
+        "    PipelineState.PIPELINE_STATE_RUNNING\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712 current state:\n",
+        "    PipelineState.PIPELINE_STATE_RUNNING\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712 current state:\n",
+        "    PipelineState.PIPELINE_STATE_RUNNING\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712 current state:\n",
+        "    PipelineState.PIPELINE_STATE_RUNNING\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712 current state:\n",
+        "    PipelineState.PIPELINE_STATE_RUNNING\n",
+        "    ...\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:AutoMLImageTrainingJob run completed. Resource name: projects/759209241365/locations/us-central1/trainingPipelines/2109316300865011712\n",
+        "    INFO:google.cloud.aiplatform.training_jobs:Model available at projects/759209241365/locations/us-central1/models/1284590221056278528"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "evaluate_the_model:migration"
+      },
+      "source": [
+        "## Evaluate the model"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "models_evaluations_list:migration,new"
+      },
+      "source": [
+        "### [projects.locations.models.evaluations.list](https://cloud.devsite.corp.google.com/ai-platform-unified/docs/reference/rest/v1beta1/projects.locations.models.evaluations/list)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "evaluate_the_model:mbsdk"
+      },
+      "source": [
+        "## Review model evaluation scores\n",
+        "After your model has finished training, you can review the evaluation scores for it.\n",
+        "\n",
+        "First, you need to get a reference to the new model. As with datasets, you can either use the reference to the model variable you created when deployed the model or you can list all of the models in your project."
```

Comment:
Add "you" to "when deployed" to make it: 

when you deployed the model...

Reply:
updated